Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -35,4 +28,56 @@ function GetVssConnection
     return $vssConnection
 }
 
+#
+# The issues, as reported by SonarQube, need to be transformed to a simpler structure that the PostComments module can consume
+# 
+function GetCommentsFromIssues
+{
+    param ([Array]$issues)
+    
+    Write-Verbose "Transforming SonarQube analysis issues to PR comments"
+    
+    $comments = New-Object System.Collections.ArrayList
+
+    foreach ($issue in $issues)
+    {
+        $priority = GetCommentPriority $issue
+
+        Assert ( ![String]::IsNullOrWhiteSpace($issue.message) ) "Internal error: the SonarQube reported issues do not have a property named 'message'"
+        Assert ( ![String]::IsNullOrWhiteSpace($issue.line) ) "Internal error: the SonarQube reported issues do not have a property named 'line'"        
+        Assert ( ![String]::IsNullOrWhiteSpace($issue.relativePath) ) "Internal error: the SonarQube reported issues do not have a property named 'relativePath'"
+        
+        $properties = @{
+            Content = $issue.message
+            Line = $issue.line
+            RelativePath = $issue.relativePath
+            Priority = $priority 
+        }
+
+        $comment = new-object PSObject -Property $properties
+
+        [void]$comments.Add($comment)
```

Comment:
Indeed, that will also do it. It seems that using [void]Foo is the suggested solution on most questions about this on StackOverflow so I'll keep it as is. There are also other places in the tasks where I used this (not under code review).